lib/events/search_limiter_test.go

```diff
@@ -105,7 +105,7 @@ func TestSearchEventsLimiter(t *testing.T) {
 				Order: types.EventOrderAscending,
 			})
 			return err == nil
-		}, 40*time.Millisecond, 5*time.Millisecond)
+		}, 80*time.Millisecond, 5*time.Millisecond)
```

Comment:
IMHO, 80ms is still quite small for CI, which can get really slow sometimes. Any issue you see if using an even bigger number like a second?